worldedit-bukkit/adapters/adapter-1_19/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R1/PaperweightGetBlocks.java

```diff
@@ -1064,22 +1064,35 @@ private void fillLightNibble(char[][] light, LightLayer lightLayer, int minSecti
         }
     }
 
-    private void setBiomesToPalettedContainer(
-            final BiomeType[] biomes,
-            PalettedContainer<Holder<Biome>> data
+    private PalettedContainer<Holder<Biome>> setBiomesToPalettedContainer(
+            final BiomeType[][] biomes,
+            final int sectionIndex,
+            final PalettedContainerRO<Holder<Biome>> data
     ) {
-        int index = 0;
-        if (biomes == null) {
-            return;
+        PalettedContainer<Holder<Biome>> biomeData;
+        if (data instanceof PalettedContainer<Holder<Biome>> palettedContainer) {
+            biomeData = palettedContainer;
+        } else {
+            LOGGER.warn(
```

Comment:
I left this as warn purposely as it's not necessarily "breaking". They should already have been/should consequently received about the history not saving

Reply:
I commented here for sake of completeness with the usage of logger#error in the diff below.

Reply:
The error is history, the warn is setting. Technically if there "error" isn't sent, it's because the edit can be undone, and the biomes return to their original state, thus the warn when setting is an issue